libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar-label.pipe.ts

```diff
@@ -0,0 +1,47 @@
+import { Pipe, PipeTransform, inject } from '@angular/core';
+
+import { SkyDatepickerCalendarInnerComponent } from './datepicker-calendar-inner.component';
+
+/**
+ * Formats date values according to formatting rules.
+ * @example
+ * ```markup
+ * {{ myDate | skyDatepickerCalendarLabel }}
+ * {{ myDate | skyDatepickerCalendarLabel:'YYYY' }}
+ * ```
+ * @internal
+ */
+@Pipe({
+  name: 'skyDatepickerCalendarLabel',
+  standalone: true,
+})
+export class SkyDatepickerCalendarLabelPipe implements PipeTransform {
+  #datepicker = inject(SkyDatepickerCalendarInnerComponent);
```

Comment:
Instead of injecting the component, could you simply use the same formatter utility the component is using?
```
#dateFormatter = new SkyDateFormatter();
```

Reply:
`dateFilter()` isn't the only thing we are getting from the component, We also pull in a default date format. Since that's the source of truth for all of the various date formats we need in the calendar, I'm inclined to keep this as is.